File: springboot_reportserversystem/src/main/java/cn/wi/pojo/Message.java
package cn.wi.pojo;

/**
 * @ProjectName: Flink_PYG
 * @ClassName: Message
 * @Author: xianlawei
 * @Description: bean对象
 * @date: 2019/9/4 21:50
 */
public class Message {
    /**
     * 发送条数
     * 发送的消息
     * 发送的时间
     */
    private int count;
    private String message;
    private Long timeStamp;

    public int getCount() {
        return count;
    }

    public void setCount(int count) {
        this.count = count;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public Long getTimeStamp() {
        return timeStamp;
    }

    public void setTimeStamp(Long timeStamp) {
        this.timeStamp = timeStamp;
    }

    @Override
    public String toString() {
        return "Message{" +
                "count=" + count +
                ", message='" + message + '\'' +
                ", timeStamp=" + timeStamp +
                '}';
    }
}

File: scala_clickstream/src/main/scala/cn.wi/pojo/Message.scala
package cn.wi.pojo

/**
 * @ProjectName: Flink_PYG 
 * @ClassName: Message
 * @Author: xianlawei
 * @Description:
 *  数据格式
 * {
 * "count": 1,
 * "message": "{\"browserType\":\"火狐\",\"categoryId\":14,\"channelId\":7,\"city\":\"America\",\"country\":\"America\",\"entryTime\":1544630460000,\"leaveTime\":1544634060000,\"network\":\"联通\",\"produceId\":11,\"province\":\"china\",\"source\":\"360搜索跳转\",\"timestamp\":1567689228665,\"userId\":7}",
 * "timeStamp": 1567689229717
 * }
 *
 * @date: 2019/9/5 20:52
 */
case class Message(count:Int,userBrowse: UserBrowse,timeStamp: Long)
